apps/docs/content/guides/auth/row-level-security.mdx

```diff
@@ -125,6 +125,20 @@ on my_table for select
 using ( true );
 ```
 
+### Protect views using `security_invoker`
+
+Views do _not_ obey RLS policies by default. This is a feature of Postgres, which automatically creates views with `security definer`.
+
+In Postgres 15 and above, you can make a view obey the RLS policies of the underlying tables by setting `security_invoker = true`.
+
+```sql
+create view
+security_invoker = true
```

Comment:
this could read a bit nicer with placeholders for the other parts of the `create view` statements had placeholders. like `<view name>` etc